taipy/gui/utils/_evaluator.py

```diff
@@ -210,15 +210,17 @@ def __evaluate_holder(self, gui: Gui, holder: t.Type[_TaipyBase], expr: str) ->
             _warn(f"Cannot evaluate expression {holder.__name__}({expr_hash},'{expr_hash}') for {expr}", e)
         return None
 
-    def evaluate_expr(self, gui: Gui, expr: str, lazy_declare: t.Optional[bool] = False) -> t.Any:
-        if not self._is_expression(expr):
+    def evaluate_expr(
+        self, gui: Gui, expr: str, lazy_declare: t.Optional[bool] = False, lambda_expr: t.Optional[bool] = False
+    ) -> t.Any:
+        if not self._is_expression(expr) and not lambda_expr:
             return expr
-        var_val, var_map = self._analyze_expression(gui, expr, lazy_declare)
+        var_val, var_map = ({}, {}) if lambda_expr else self._analyze_expression(gui, expr, lazy_declare)
         expr_hash = None
         is_edge_case = False
 
         # The expr_string is placed here in case expr get replaced by edge case
-        expr_string = 'f"' + expr.replace('"', '\\"') + '"'
+        expr_string = expr if lambda_expr else 'f"' + expr.replace('"', '\\"') + '"'
```

Comment:
That 'replace' is a bit weak in my opinion.
What if the expression contains the string "\""?
That could be another issue for the time being, honestly.

Reply:
We'll deal with it if/when it appears ...
This is quite a complicated piece of code